test/extended/deployments/deployments.go

```diff
@@ -68,12 +68,12 @@ var _ = g.Describe("deploymentconfigs", func() {
 				case n < 0.7:
 					// cancel any running deployment
 					e2e.Logf("%02d: cancelling deployment", i)
-					if out, err := oc.Run("deploy").Args("dc/deployment-simple", "--cancel").Output(); err != nil {
+					if out, err := oc.Run("rollout").Args("cancel", "dc/deployment-simple").Output(); err != nil {
 						// TODO: we should fix this
```

Comment:
@kargakis i think I like the PATCH approach... we should get it merged in upstream... i will spawn an issue and do it as follow up.